Run track p2

P-Schumacher opened the pull request.

Vittorio-Caggiano commented on myosuite/envs/myo/assets/leg/myoosl_runtrack_p2.xml:
```diff
@@ -0,0 +1,47 @@
+
+        <camera name="side_view" pos="-3 0 2.25" xyaxes="0 -1 0 1 0 2" mode="trackcom"/>
+
+        <!-- ======= MyoAgent ======= -->
```
not sure if we standardized on this but Agent might indicate more the controller. I would say MyoActor or MyoBody to indicate only the body/actuation side

P-Schumacher replied:
has been addressed

Vittorio-Caggiano reviewed (approved):
lgtm. 
just two suggestions:
1. [major] add a test for the application (and reproducibility) of the terrains
2. [minor] change the indication of the composite msk model from MyoAgent to something else

P-Schumacher commented:
> lgtm. just two suggestions:
> 
> 1. [major] add a test for the application (and reproducibility) of the terrains
> 2. [minor] change the indication of the composite msk model from MyoAgent to something else

Changed the name in the xml and added a `test_heightfields.py` file.

Vittorio-Caggiano commented:
> > lgtm. just two suggestions:
> > 
> > 1. [major] add a test for the application (and reproducibility) of the terrains
> > 2. [minor] change the indication of the composite msk model from MyoAgent to something else
> 
> Changed the name in the xml and added a `test_heightfields.py` file.

thanks Pierre!
